pkg/render/compliance_test.go

```diff
@@ -1164,16 +1164,25 @@ var _ = Describe("compliance rendering tests", func() {
 
 			tenantAResources, _ := tenantACompliance.Objects()
 
-			rbTenantA := rtest.GetResource(tenantAResources, render.ComplianceServerServiceAccount, "", rbacv1.GroupName, "v1", "ClusterRoleBinding").(*rbacv1.ClusterRoleBinding)
-			Expect(rbTenantA.RoleRef.Kind).To(Equal("ClusterRole"))
-			Expect(rbTenantA.RoleRef.Name).To(Equal(render.ComplianceServerServiceAccount))
-			Expect(rbTenantA.Subjects).To(ContainElements([]rbacv1.Subject{
-				{
-					Kind:      "ServiceAccount",
-					Name:      render.ComplianceServerServiceAccount,
-					Namespace: tenantANamespace,
-				},
-			}))
+			expectedClusterRoleBindings := []string{render.ComplianceBenchmarkerServiceAccount,
```

Comment:
nit, but if you're spreading across multiple lines it should be formatted like this:

```
[]string{
  render.ComplianceBenchmarkerServiceAccount,
}
```